src/AppBundle/Procuration/ProcurationProxyMessageFactory.php

```diff
@@ -40,4 +41,17 @@ public function createProxyFoundMessage(Adherent $procurationManager, Procuratio
 
         return $message;
     }
+
+    public function createProxyReminderMessage(ProcurationRequest $request): ProcurationProxyReminderMessage
+    {
+        $url = $this->urlGenerator->generate('app_procuration_my_request', [
+            'id' => $request->getId(),
+            'token' => $request->generatePrivateToken(),
+        ]);
+
+        $message = ProcurationProxyReminderMessage::create($request, $request->getFoundProxy(), $url, $request->getFoundBy());
```

Comment:
There is no need to pass the extra arguments which depend on the request object, just pass the request and use it to get its values in the static method instead, or is there other usages I'm missing?